uber/model_checks.py

```diff
@@ -118,8 +119,11 @@ def group_money(group):
 
 
 def _invalid_phone_number(s):
-    if not s.startswith('+'):
-        return len(re.findall(r'\d', s)) != 10 or re.search(c.SAME_NUMBER_REPEATED, re.sub(r'[^0-9]', '', s))
+    try:
+        parsed = phonenumbers.parse(s, 'US')
```

Comment:
Forgive me, I am not super familiar with the `phonenumbers` library. Will this correctly handle international numbers? Looks like the old code would consider any number that started with `+` to be international and allow it. But what we really want is to actually validate international numbers.

Reply:
The second parameter is simply the default country to assume if no ``+`` is present, e.g. if I give it something like ``703-451-2125``.  The function will treat any number with a leading ``+`` as an international number and do the right thing based on the country code.

Reply:
Nice!

Reply:
My understanding is that the second parameter is more like a fallback option. When a leading `+` is provided, then that means the country code was also provided, so it can validate accordingly. Otherwise, there is no way to know what country the number belongs to, so the second parameter instructs it to assume it is a US number.